dimod/generators/anti_crossing.py

```diff
@@ -12,89 +12,102 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
-from dimod import BinaryQuadraticModel
+from dimod.binary.binary_quadratic_model import BinaryQuadraticModel
+from dimod.vartypes import Vartype
 
 __all__ = ['anti_crossing_clique', 'anti_crossing_loops']
 
 
-def anti_crossing_clique(num_variables):
-    """Anti crossing problems with a single clique.
+def anti_crossing_clique(num_variables: int) -> BinaryQuadraticModel:
+    """Generate an anti crossing problem with a single clique.
 
-    Given the number of variables, the code will generate a clique of size
-    num_variables/2, each variable ferromagnetically interacting with a partner 
-    variable with opposite bias. A single variable in the cluster will have 
-    no bias applied.
+    Let ``N = num_variables // 2``. This function returns a binary
+    quadratic model with variables `[0, N)` forming a ferromagnetic clique.
```

Comment:
On first read it seems that this consists of the entire BQM. Perhaps, "This function returns a binary quadratic model where half the variables, ``[0, N)``,  form a ferromagnetic clique, with each variable, ``v``, also ferromagnetically interacting with one variable, ``v+N``, of the remaining half of the variables, ``[N, 2*N)``."